scripts/codegen.py

```diff
@@ -70,8 +70,10 @@ def write_new_data(self, relative_path: str, content: str):
 @click.option(
     '--generator',
     default='JAVA',
-    type=click.Choice(GENERATORS.keys(), case_sensitive=False),
-    help='What code generator to run')
+    help='What code generator to run.  The choices are: '+'|'.join(GENERATORS.keys())+'. ' +
+         'When using custom, provide the plugin path using ' +
+         '`--generator custom:<path_to_plugin>` syntax. The plugin path shall contain a package ' +
```

Comment:
the module logic was updated to figure out the module name. We probably should update the comment too.